plugin/plugin.go

```diff
@@ -21,6 +24,17 @@ func (f *funcPlugin) Name() string {
 }
 
 func (f *funcPlugin) Execute(args []string) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	sigs := make(chan os.Signal, 1)
```

Comment:
@rhuss @dsimansk
I wonder how this would play with kn plugin system.
Would the plugin catch the signal, or kn would "steal" it?

Reply:
this seems to work

Reply:
looks like `kn` is not explicitly handling sigter or sigint